### PR TITLE
Improve flow in onboarding screen #4655

### LINKED ISSUE #4655: Improve flow in onboarding screen
**Describe the bug**
When we click next button in onboarding screen, focus will remain in next button instead of announcing current screen.

**To Reproduce**
1. Clear app's data if Oppia app is already installed
2. Turn on Talkback.
3. Click on 'Next Button'.

**Expected behavior**
When we click next button, talkback reader should read screen name. 

**Demonstration**

https://user-images.githubusercontent.com/43074241/195674699-2c58a48a-f04f-47ef-8acc-ca3476da5c57.mp4

**Environment**
- Device/emulator being used: Realme GT Master Edition
- Android or SDK version (e.g. Android 5 or SDK 21): Android 12
- App version (you can get this through system app settings or via the admin controls menu in-app): 1.0

### PR DESCRIPTION
**Pull request by iampradeep-hr**
## Explanation
  - Improve flow in onboarding screen #4655
  - "Fixes #4655:" improved the talkback flow in onborading
  

## For UI-specific PRs only

https://user-images.githubusercontent.com/81664507/204102872-a30b751f-e92f-4a6c-8c84-f357e9a00b1f.mp4

### CODE REVIEW

**Comment by pratyaksh1610** [comment]
@iampradeep-hr Looks like you have not signed the CLA , refer [this](https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android#onboarding-instructions) to get started.

**Comment by iampradeep-hr** [comment]
> @iampradeep-hr Looks like you have not signed the CLA , refer [this](https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android#onboarding-instructions) to get started.

Hi @pratyaksh1610 , Thanks a lot. I am new to this. What is CLA sheet and how to sign it up? 
I don't see any sign up option that page.

**Comment by pratyaksh1610** [comment]
@iampradeep-hr You will need to fill the form to get started (refer to the first point).

**Comment by iampradeep-hr** [comment]
> @iampradeep-hr You will need to fill the form to get started (refer to the first point).

Ok, Thanks a lot. I submitted the form.

**Comment by BenHenning** [comment]
Once the CI checks are done, you can assign it to me to take a review pass @iampradeep-hr by replying in a comment with 'PTAL @BenHenning'.

**Comment by iampradeep-hr** [comment]
PTAL @BenHenning

File: app/src/main/java/org/oppia/android/app/onboarding/OnboardingFragmentPresenter.kt [hunk 1]
```diff
@@ -82,6 +83,15 @@ class OnboardingFragmentPresenter @Inject constructor(
           positionOffset: Float,
           positionOffsetPixels: Int
         ) {
+          super.onPageScrolled(position, positionOffset, positionOffsetPixels)
+          binding.root.performAccessibilityAction(
+            AccessibilityNodeInfoCompat.ACTION_ACCESSIBILITY_FOCUS,
+            null
+          )
+          binding.onboardingSlideViewPager.performAccessibilityAction(
```
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/onboarding/OnboardingFragmentPresenter.kt:91]: Why does this need to be done on both the root and the view pager?

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @iampradeep-hr. Took a pass. I left one comment, but otherwise:
- Please update the PR description & title to be consistent with other PRs (I think you erased parts of the PR description). See #4685 for an example of both.
- We should add tests for the change to make sure the event is being sent when expected.

**Pull request closed without merging**